README.md

```diff
@@ -1,5 +1,6 @@
 # Standardize PR Comment
```

Comment:
self-test should be successful

Reply:
:partying_face: https://github.com/Nebulaworks/action-standardize-pr-comment/actions/runs/5194962979/jobs/9367198868